starter-code/css/main.css

```diff
@@ -0,0 +1,66 @@
+* {
+  margin: 10px;
+  font-family: 'Lato', sans-serif;
+  text-align: left;
+}
+body {
+  text-align: left;
+}
+h1 {
+  font-size: 45px;
+  text-align: left;
+  font-weight: 100;
+  margin-left: 0;
+}
+h2 {
+  font-size: 30px;
+  font-weight: lighter;
+  letter-spacing: -1px;
+}
+h2 a {
+  color: #000000;
+  font-weight: 550 !important;
+  margin-left: 0;
+}
+h3 {
+  font-size: 35px;
+  font-weight: lighter;
+  color: #E2DED6;
+}
+h4 {
+  float: right;
+  padding: 15px 0px 10px 0px;
+}
+hr {
+  color: #E7E3DB;
+}
+a {
+  text-decoration: none;
+  color: #F9CA72;
+}
+p {
+  text-align: left;
+}
+nav a {
+  text-decoration: underline;
+  color: #E2DED6;
+  font-size: 20px;
+}
+nav {
+  padding: 10px 0px 50px 0px;
+  margin-left: 0;
+}
+span {
+  color: #E2DED6;
+  font-size: 95px;
+  font-weight: 500;
+}
+footer a {
+  color: #F9CA72;
+  font-size: 14px;
+  margin: 0px 2px 0px 2px;
```

Comment:
Another shorthand way to write this is `margin: 0 2px`. This will set the top and bottom to 0px and the left and right to 2px.